doc/cli/operator-sdk_alpha_olm.md

```diff
@@ -15,7 +15,9 @@ Manage the Operator Lifecycle Manager installation in your cluster
 ### SEE ALSO
 
 * [operator-sdk alpha](operator-sdk_alpha.md)	 - Run an alpha subcommand
+* [operator-sdk alpha olm down](operator-sdk_alpha_olm_down.md)	 - Tears down the operator with Operator Lifecycle Manager
 * [operator-sdk alpha olm install](operator-sdk_alpha_olm_install.md)	 - Install Operator Lifecycle Manager in your cluster
 * [operator-sdk alpha olm status](operator-sdk_alpha_olm_status.md)	 - Get the status of the Operator Lifecycle Manager installation in your cluster
 * [operator-sdk alpha olm uninstall](operator-sdk_alpha_olm_uninstall.md)	 - Uninstall Operator Lifecycle Manager from your cluster
+* [operator-sdk alpha olm up](operator-sdk_alpha_olm_up.md)	 - Launches the operator with Operator Lifecycle Manager
```

Comment:
```suggestion
* [operator-sdk alpha olm install](operator-sdk_alpha_olm_up.md)	 -  Install the OLM project bundle in your cluster to be checked with OLM catalogue. 
```

Reply:
This doc is auto-generated correctly, hence the passing tests.